tests/test_parser.py

```diff
@@ -86,38 +86,34 @@ def setup_class(cls):
          ValueQuery(Value('$e^{+}e^{-}$'))),
 
         # Basic keyword:value
-        ("foo:bar",
-         KeywordOp(Keyword('foo'), Value('bar'))),
-        ("foo: bar",
-         KeywordOp(Keyword('foo'), Value('bar'))),
-        ("foo: 2004",
-         KeywordOp(Keyword('foo'), Value('2004'))),
-        ("999: bar",
-         KeywordOp(Keyword('999'), Value('bar'))),
-        ("999C5: bar",
-         KeywordOp(Keyword('999C5'), Value('bar'))),
```

Comment:
These are no longer working? I assume if we explicitly add these as keywords they will again... These were MARC based queries :)